### DISCUSSION TITLE
Backup & rrstore

### DISCUSSION THREAD

**Discussion by ghost** [BlackArch/blackarch#3054] [Q&A] [answered]
I would like to know how would one do a full "backup" and "restore" of BlackArch?
I am putting a lot of work into my build and if something gets jacked up I would like to be able to go back to a clean/stable image.
I have a 256gb USB stick to use for the backup media.
Any help ideas would be great.

Thanks

**Comment by Edu4rdSHL** [comment, marked as answer]
Hi, there are several options, see https://wiki.archlinux.org/index.php/System_backup

**Reply by ghost** [reply]
Thank you, I'll check them out

**Reply by Edu4rdSHL** [reply]
Please mark the comment as answer if this solved your question.

**Comment by ghost** [comment]
Yes I will
